operations/IT/hardware-buy-back-policy.md

```diff
@@ -0,0 +1,22 @@
+# Hardware Buy Back Policy
+
+Staff members have the option to buy back their existing hardware (laptop, desktop, phone, tablet etc.) either when it gets refreshed for a new one (which is on average, 3 years from purchase date - not necessarily 3 years of employment), or when the staff member is offboarding. 
+
+If the staff member has completed one calendar year or more at Mattermost at the time of offboarding, they can opt to keep their device at no cost. If the staff member hasn't completed one calendar year at the time of offboarding or is receiving an early hardware refresh, they have the option to purchase their hardware from Mattermost at current market value.
```

Comment:
@TQuock If staff members would like to know what that cost will be effectively, shall we say they can reach out to Finance? To IT?

Reply:
@DSchalla All questions regarding the buyback policy can be re-directed to me, I'd like to keep the main point of contact being IT for all incoming questions. I'll communicate via the private buy back policy channel (to be created) if there is any clarification needed from other teams (in this case Finance).

Reply:
@danielsischy - will this run through Freshworks as a request as well or does will this be channel driven? Just wondering how Finance will ingest

Reply:
@TQuock Because this program may periodically require input from Security, Finance, IT, & HR, I believe this should be channel driven. I'll create tickets whenever hardware is granted to an employee, but that would mostly be for auditing purposes.